service/src/main/java/bio/terra/workspace/service/job/JobService.java

```diff
@@ -143,11 +144,9 @@ protected <T> T submitAndWait(
 
   public void waitForJob(String jobId) {
     try {
-      int pollSeconds = jobConfig.getPollingIntervalSeconds();
-      int pollCycles = jobConfig.getTimeoutSeconds() / jobConfig.getPollingIntervalSeconds();
```

Comment:
It looks like this is the only place `getTimeoutSeconds` and `getPollingIntervalSeconds` config values are used - can you also remove those properties from JobConfiguration/`application.yml`/helmfile configuration (might be another ticket)?